docs/cloud/cloud-org/cloud-org-upgrade-to-paid.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Upgrade your organization to paid
+layout: default
+parent: Manage organization
+grand_parent: Cloud
+nav_order: 4
+---
+
+# How do I upgrade my organization to paid?
+
+{: .no_toc }
+
```

Comment:
Need some kind of information regarding payments here, such as:

* how the payments are secured
* is credit-card held by FeatureBase or by third-party system
* is there a payments policy or date/time when payments are withdrawn (e.g., is it on payment, does the user get to choose when payments are made? are the pre or post-paid?)
* Any other financial information relevant to payments

Reply:
FYI, according to the payments page "shapes are billed hourly and paid monthly". So need to confirm this is still the case.

Reply:
how the payments are secured - nah they can contact us for this
is credit-card held by FeatureBase or by third-party system - nah they can contact us for this
is there a payments policy or date/time when payments are withdrawn (e.g., is it on payment, does the user get to choose when payments are made? are the pre or post-paid?) - added
Does the user/org get issued with payment receipts which they can then add into their own accounting system? -added
Any other financial information relevant to payments - added

confirmed it's the same billing

Reply:
Hi @gthrone25 
> how payments are secured
This should be provided somewhere that's easy to find because there's been so many data breaches lately, customers should be informed. And if it was me, I'd be wondering if I'm throwing my ccard into an unsecured system.
> where ccard is held > there appear to be some kind of regulations around this -- see here for some background info: https://www.pcidssguide.com/pci-requirements-for-storing-credit-card-information/

Reply:
Plus I just found out there's a way to have Stripe handle the bank details. Why not just say that?

Reply:
Every db doc site I've looked at does not include this info, and we're going the same way. If people want to purchase and have questions, they will contact us, which is what we want. We can give them all of these answers then